cmake/external/extensions.cmake

```diff
@@ -21,6 +21,21 @@ if (onnxruntime_REDUCED_OPS_BUILD)
   set(OCOS_ENABLE_SELECTED_OPLIST ON CACHE INTERNAL "")
 endif()
 
+if (onnxruntime_WEBASSEMBLY_DEFAULT_EXTENSION_FLAGS)
+  set(OCOS_ENABLE_SPM_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_GPT2_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_WORDPIECE_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_BERT_TOKENIZER ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_TF_STRING ON CACHE INTERNAL "")
+  set(SPM_USE_BUILTIN_PROTOBUF OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_STATIC_LIB ON CACHE INTERNAL "")
+  set(OCOS_ENABLE_BLINGFIRE OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_OPENCV OFF CACHE INTERNAL "")
+  set(OCOS_ENABLE_CV2 OFF CACHE INTERNAL "")
```

Comment:
_CV2 Replaced _OpenCV, did you intentionally to keep both of them for multiple versions?

Reply:
I saw both in the extension cmake so I thought play safe :).
I can remove _OpenCV